src/typechecker/FStar.TypeChecker.Util.fst

```diff
@@ -51,7 +51,7 @@ let report env errs =
 (* Unification variables *)
 (************************************************************************)
 let new_implicit_var reason r env k =
-  new_implicit_var_aux reason r env k Strict None
```

Comment:
Only changes in `TcUtil` are for applying the indexed effects combinators (and some rearrangement).